tools/cpp/windows_cc_configure.bzl

```diff
@@ -93,6 +93,36 @@ def find_vc_path(repository_ctx):
     _auto_configure_warning_maybe(repository_ctx, "'BAZEL_VC' is not set, " +
                                                   "start looking for the latest Visual C++ installed.")
 
+    if "BAZEL_VC_TOOL" in repository_ctx.os.environ:
+        _auto_configure_warning_maybe(repository_ctx, "'BAZEL_VC_TOOL' is set, " +
+                               "but 'BAZEL_VC' and 'BAZEL_VS' are not. " + 
+                               "This is not recomended and can " +
+                               "lead to incorrect VC++ detection and configuration.")
+        vc_tool_path = repository_ctx.os.environ["BAZEL_VC_TOOL"]
+        # As neither 'BAZEL_VC' nor 'BAZEL_VS' are set, we assume BAZEL_VC_TOOL looks like that: %VS_PATH%\VC\<path_to_tool>
+        # path_to_tool is different for different VS versions, but we don't care
+        # We DO NOT search in any other places as we can easily 
+        # get inconsistancy with VC++ tools and VCVARSALL.BAT (assuming multiple versions of VS are installed)
+        vc_tool_path = vc_tool_path.replace('\\', '/').strip()
+        vc_pos = vc_tool_path.rfind('/VC/')
+        if vc_tool_path == 'VC' or vc_tool_path.endswith('/VC'):
```

Comment:
What are valid `BAZEL_VC_TOOL` values?

If neither `BAZEL_VC` nor `BAZEL_VS` are defined, then these envvar names should not be inside `BAZEL_VC_TOOL` either, right?

IIUC, only absolute paths like `<vs_path>\VC\<path_to_tool>` are OK. If so, the logic can be simplified:
```
vc_tool_path = vc_tool_path.strip().replace('\\', '/')
parts = vc_tool_path.rsplit('/VC/', 1)
if len(parts) == 2:
  vc_tool_path = parts[1]
```
Does that look good?